app/javascript/packs/application.js

```diff
@@ -1,13 +1,14 @@
-// This file is automatically compiled by Webpack, along with any other files
-// present in this directory. You're encouraged to place your actual application logic in
-// a relevant structure within app/javascript and only use these pack files to reference
-// that code so it'll be compiled.
-
-import Rails from "@rails/ujs"
-import Turbolinks from "turbolinks"
-import * as ActiveStorage from "@rails/activestorage"
-import "channels"
-
-Rails.start()
-Turbolinks.start()
-ActiveStorage.start()
+import "core-js/stable"
+import "regenerator-runtime/runtime"
+import "materialize-css/dist/js/materialize"
+
+import "stylesheets/application"
+
+const images = require.context('../images', true)
+const imagePath = (name) => images(name, true)
+
+require("@rails/ujs").start()
+
+document.addEventListener('DOMContentLoaded', () => {
+   M.AutoInit();
+ });
```

Comment:
```suggestion
document.addEventListener('DOMContentLoaded', () => {
  M.AutoInit();
});
```
відступи мають бути 2 пункти, собі в едіторі налаштуй це